src/test/java/seedu/cafectrl/command/AddOrderCommandTest.java

```diff
@@ -123,45 +113,23 @@ public void showTotalCost(String dollarCost) {
         Command addOrderCommand = new AddOrderCommand(orderChickenRice, ui, pantry, orderList, menu);
         addOrderCommand.execute();
 
-        Command addOrderCommand2 = new AddOrderCommand(orderChickenCurry, ui, pantry, orderList, menu);
-        addOrderCommand2.execute();
+        String actualOutput = baos.toString().trim();
+        System.setOut(originalOut);
 
-        String actualOutput = String.join(",", commandOutput);
+        String expectedOutput = "I'm busy crafting your selected dish in the virtual kitchen of your dreams. "
```

Comment:
Perhaps you should use the Strings stored in Messages.java instead of "magic strings"